functions/SharedFunctions.ps1

```diff
@@ -1498,6 +1498,11 @@ Function Update-SqlPermissions
 		
 		if ($destdb -ne $null)
 		{
+			if (!$destdb.IsAccessible)
+			{
+				Write-Output "Database is not accessible. Skipping"
```

Comment:
thoughts on outputting which database?
`Write-Output "Database $destdb is not accessible. Skipping"`